lib/index.js

```diff
@@ -45,9 +45,8 @@ module.exports = {
     this._super.init && this._super.init.apply(this, arguments);
 
     if ('ember' in this.project.bowerDependencies()) {
-      // TODO: move this to a throw soon.
-      this.ui.writeWarnLine(
-        'Ember.js is now provided by node_module `ember-source`, please remove it from bower'
+      throw new SilentError(
+        'Ember.js is now provided by the `ember-source` node module, please remove `ember` from your `bower.json` file.'
       );
     }
```

Comment:
Can we remove the whole `if ('ember' in this.project.bowerDependencies()) {` block? cc @rwjblue ?

Reply:
Sure, let me know if I should remove.